src/lib.rs

```diff
@@ -1227,6 +1227,28 @@ where
     }
 }
 
+impl<K, V, S> PartialEq for FlurryHashMap<K, V, S>
+where
+    K: Sync + Send + Clone + Eq + Hash,
+    V: Sync + Send + PartialEq,
+    S: BuildHasher,
+{
+    fn eq(&self, other: &Self) -> bool {
+        let guard = epoch::pin();
+
```

Comment:
I believe you also have to check `self.len() == other.len()` here

Reply:
Yep, I know, but there's currently no `len` method. 
I would need to  create two iterators for both maps to compare the length atm

Reply:
The `len` method is coming in https://github.com/jonhoo/flurry/pull/6, so I guess we should just wait for that. Or just copy that part of the change into here :)

Reply:
Okay, I will wait for #6 until this PR is ready, otherwise I will copy the part

Reply:
Should just be a couple of minutes and then I'll merge it :)

Reply:
All right, it's landed on `master`!